### PR TITLE
chore: bump DS 2.0.0-rc.9

### PR DESCRIPTION
**Pull request by madhurisandbhor**
### What does it do?
Bump DS 2.0.0-rc.9

### Related issue(s)/PR(s)

Fixes: https://github.com/strapi/strapi/issues/20762, https://github.com/strapi/strapi/issues/20767, https://github.com/strapi/strapi/issues/20776, https://github.com/strapi/strapi/issues/20787, https://github.com/strapi/strapi/issues/20761, https://github.com/strapi/strapi/issues/20803

### CODE REVIEW

**Comment by lucasboilly** [comment]
@madhurisandbhor it looks really better! 

The only issue that seems critical is the responsive version being triggered a bit too soon, you can see how it looks like in this screenshot with Arc's navigation bar expanded on a 13-inches MacBook Pro. I guess it could be worse on smaller computers.

CleanShot 2024-08-13 at 11 11 20@2x

Then there's some other smaller issues that can certainly be solved later:
- The checkboxes are still using the old icons
CleanShot 2024-08-13 at 10 14 29@2x
- The container of the avatar in the main nav should have a fixed height to make it a square (already mentioned [here](https://www.notion.so/strapi/The-container-of-the-Avatar-in-the-MainNav-should-be-a-square-56x56-8827d78abbfa47d8a5bb9966d0cf756e?pvs=4) in the Notion document of the last blitz session but it seems like it has been affected by the last DS changes)
CleanShot 2024-08-13 at 10 30 40@2x
- The cross icon si too small in the tags
CleanShot 2024-08-13 at 10 41 14@2x
- We should give a fixed heights to the selects because their height changes depending on their content
CleanShot 2024-08-13 at 10 06 44@2x
- There's a white space in the disabled version of the TextArea input
CleanShot 2024-08-13 at 10 06 55@2x
- The left icon is not aligned in the block selector in the Blocks field
CleanShot 2024-08-13 at 10 07 20@2x
- The icon buttons in Bulk Publish are not squares
CleanShot 2024-08-13 at 10 33 49@2x
- The icon buttons in the RTE are too big
CleanShot 2024-08-13 at 10 07 07@2x

**Comment by lucasboilly** [comment]
2 other very small issues:

- It seems the fix you did for the colors of the icon in the icon buttons doesn't work anymore
- The space above the popover in the simple menu is too big, but it was all good in the preview

CleanShot 2024-08-13 at 12 36 22@2x

**Comment by madhurisandbhor** [comment]
Hello @lucasboilly, @oiorain  I have pushed some of the quick fixes on CMS, but most of the issues mentioned are not part of DS `2.0.0-rc.9` change, so this PR should be merged once CMS fixes are working.

1. Responsive version being triggered a bit too soon - fixed in DS, https://github.com/strapi/design-system/pull/1775
2. The container of the avatar in the main nav - fixed in CMS
3. The icon buttons in Bulk Publish are not squares - fixed in CMS
4. The icon buttons in the RTE are too big - fixed in CMS

Needs to be fixed in DS : 

1. The checkboxes are still using the old icons
2. The cross icon si too small in the tags (MultiSelect)
3. We should give a fixed heights to the selects because their height changes depending on their content
4. There's a white space in the disabled version of the TextArea input
5. The left icon is not aligned in the block selector in the Blocks field (Select)

Needs to be fixed in CMS : 
1. It seems the fix you did for the colors of the icon in the icon buttons doesn't work anymore
2. The space above the popover in the simple menu is too big, but it was all good in the preview

File: packages/core/admin/admin/src/components/MainNav/NavUser.tsx [hunk 1]
```diff
@@ -13,15 +13,15 @@ export interface NavUserProps extends ButtonProps {
...
- */
 const MenuTrigger = styled(Menu.Trigger)`
   height: 100%;
   border-radius: 0;
   border-width: 1px 0 0 0;
   border-color: ${({ theme }) => theme.colors.neutral150};
   border-style: solid;
+  padding: ${({ theme }) => theme.spaces[3]};
+  // padding 12px - 1px border width
+  padding-top: 11px;
```
[the remark this answers is not in the document]
Comment (simotae14) [reply packages/core/admin/admin/src/components/MainNav/NavUser.tsx:24]: If we remove it there is a difference
Schermata 2024-08-22 alle 11 10 22
As you can see we don't have space between the icon and the bottom border

I prefer to stay with Madhuri's solution

File: packages/core/content-manager/admin/src/pages/ListView/components/BulkActions/PublishAction.tsx [hunk 2]
```diff
@@ -240,30 +240,32 @@ const SelectedEntriesTableContent = ({
...
+                        },
+                        { number: index + 1 }
+                      ),
+                    }
+                  )}
+                  target="_blank"
+                  marginLeft="auto"
+                >
+                  <Pencil width={'1.6rem'} height={'1.6rem'} />
+                </IconButton>
```
[the remark this answers is not in the document]
Comment (simotae14) [reply packages/core/content-manager/admin/src/pages/ListView/components/BulkActions/PublishAction.tsx:267]: Done, thank you!

File: packages/core/content-releases/admin/src/components/ReleaseListCell.tsx [hunk 3]
```diff
@@ -98,12 +90,16 @@ const ReleaseListCell = ({ documentId, model }: ReleaseListCellProps) => {
   return (
     <Popover.Root>
       <Popover.Trigger>
-        <StyledButton
+        <Button
           variant="ghost"
           onClick={(e: React.MouseEvent<HTMLElement>) => e.stopPropagation()}
-          endIcon={<CaretDown />}
+          endIcon={releases.length > 0 ? <CaretDown width="1.2rem" height="1.2rem" /> : null}
```
[the remark this answers is not in the document]
Comment (simotae14) [reply packages/core/content-releases/admin/src/components/ReleaseListCell.tsx:96]: I added a TODO for the next iterations on the DS

File: packages/core/content-manager/admin/src/pages/ListView/components/TableCells/Relations.tsx [hunk 4]
```diff
@@ -68,18 +67,23 @@ const RelationMultiple = ({ mainField, content, rowId, name }: RelationMultipleP
...
-              id: 'content-manager.containers.list.items',
-              defaultMessage: '{number, plural, =0 {items} one {item} other {items}}',
-            },
-            { number: content.count }
-          )}
-        </Flex>
-      </MenuTrigger>
+      <Menu.Trigger onClick={(e) => e.stopPropagation()}>
+        <Typography
+          style={{ maxWidth: '252px', cursor: 'pointer' }}
```
[the remark this answers is not in the document]
Comment (simotae14) [reply packages/core/content-manager/admin/src/pages/ListView/components/TableCells/Relations.tsx:72]: Yeah, I don't see the point to define the maxWidth, we can try to remove it and if we find some issues in future we can re add it

File: packages/core/content-manager/admin/src/pages/ListView/components/BulkActions/PublishAction.tsx [hunk 5]
```diff
@@ -240,30 +240,32 @@ const SelectedEntriesTableContent = ({
...
+                        {
+                          id: 'content-manager.components.ListViewHelperPluginTable.row-line',
+                          defaultMessage: 'item line {number}',
+                        },
+                        { number: index + 1 }
+                      ),
+                    }
+                  )}
+                  target="_blank"
+                  marginLeft="auto"
```
Comment (jhoward1994) [inline packages/core/content-manager/admin/src/pages/ListView/components/BulkActions/PublishAction.tsx:264]: If 'remember me' isn't enabled new tabs open to the log in page, which makes
opening the different locales in new tabs a bit frustrating.

Do we want to open locales in _blank ?

Comment (Feranchz) [reply packages/core/content-manager/admin/src/pages/ListView/components/BulkActions/PublishAction.tsx:264]: If I remember well this is intended, the idea is users open the BulkPublish modal and then they can open multiple tabs to edit entries, if we don't use _blank you lose the state of the bulkPublish modal when you go to the other page

So we decided to do it like this even if users without "remember me" are going to have problems

**Review by Feranchz — APPROVED** [review]
Everything looks good but there is a front test failing 🤔

**Pull request merged**